modules/virt-customizing-storage-profile.adoc

```diff
@@ -67,3 +67,37 @@ status:
 <2> The `volumeMode` that you select.
 +
 After you save your changes, the selected values appear in the storage profile `status` element.
+
+== Setting a default cloning strategy using a storage profile
+
+You can use storage profiles to set a default cloning method for a storage class, creatng a _cloning strategy_. Setting cloning strategies  can be helpful, for example, if your storage vendor only supports certain cloning methods. It also allows you to select a method that limits resource usage or maximizes performance.
+
+Cloning strategies can be specified by setting the `cloneStrategy` attribute in a storage class to one of these values:
+
+* `snapshot` - The default cloning method, using smart cloning. Smart cloning quickly and efficiently creates a clone by cloning the data volume PVC to another PVC.
```

Comment:
Perhaps state this like the following: "This method will be used by default when snapshots are configured"

I would like to move away from the term "smart cloning" for a couple of reasons.  a snapshot-based clone and a csi clone are both "smart" in that they use advanced capabilities of the storage array to efficiently perform the clone.  We don't call host-assisted clone "dumb" or "not smart".  Sometimes host-assisted clone is the smartest option because it's the only option that will work.

We can say that this cloning strategy uses a temporary volume snapshot to clone the volume.  The storage provisioner must support CSI snapshots.

Reply:
Done. Changed to: 

* `snapshot` - This method will be used by default when snapshots are configured. This cloning strategy uses a temporary volume snapshot to clone the volume. The storage provisioner must support CSI snapshots.

@aglitke - If you like, create a story about removing the "smart clone" verbiage in the storage doc for Virt in 4.10 or a bug if you would like it eliminated sooner. Thanks!